app/src/main/java/com/github/polybooks/utils/SaleUtil.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.polybooks.utils
+
+import com.github.polybooks.core.database.interfaces.SaleOrdering
+
+val saleOrderingTextValues = mapOf<SaleOrdering, String>(
```

Comment:
Shouldn't those go in the android Strings thing?
or directly in the SaleOrdering enum, like the fieldName in the BookFields enum

Reply:
Yes, about the string thing you are refering to I know it is something used in activity layouts, not sure if it is common to use it in the code as well but we can totally do that. 

Oh yeah I would probably make more sense to put it with the enum directly